var/spack/repos/builtin/packages/automake/package.py

```diff
@@ -2,13 +2,10 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
-from spack import *
-
-import os
 import re
 
 
+@detectable
```

Comment:
This will need to be mentioned in the docs

Reply:
@adamjstewart @scheibelp Docs have been extended in e41afff Let me know what you think of it.